README.md

```diff
@@ -2,95 +2,81 @@
-  * Refresh [http://localhost/](http://localhost/)
-* Permanently:
-  * Delete the contents of `frontend/build` directory
-  * Comment out the `try_files` line in nginx.conf as per above by placing a `#` before `try_files`
-  * Restart the web container
-    * `docker compose restart web`
-
-## Development à la Carte
-
-### Front End
-
-#### Setup
-* ```cd gdx-agreements-tracker/frontend```
-* ```yarn install```
-* Commands
-    * ```yarn start``` - starts front end app on localhost:3000
-    * ```yarn build``` - builds for production deployment
-
-#### Build
-* ```cd gdx-agreements-tracker-front-end/frontend```
-* ```docker build -t gdx-agreements-tracker-front-end:latest .```
-* Test build by running ```docker run -p 8081:80 --rm gdx-agreements-tracker-front-end```
-
-
-### Back End
-
-#### Setup
-* ```cd gdx-agreements-tracker/backend```
-* ```npm i```
-* Commands
-  * ```npm run start``` - to start api server on localhost:8080
-  * ```npm run test``` - to run tests.
-
-#### Build
-* ```cd gdx-agreements-tracker/backend```
-* ```docker build -t gdx-agreements-tracker-api:latest .```
+
+1.  Clone the repo to your local development system:
+    ```
+    git clone https://github.com/bcgov/gdx-agreements-tracker.git
+    ```
+---
+
+2. Install dependencies for the backend. 
+    * Open a new terminal in the directory of the repo you just cloned locally and run the following commands: 
+        ```
+        cd /backend
+        npm i
+        ``` 
+---
+
+3. Now install dependencies for the frontend. 
+    * Open a new terminal in the directory of the repo you just cloned locally and run the following commands: 
+        ```
+        cd /frontend
+        yarn i
+        ```
+---
+
+4. Run the seeders and migrations for the local development database.
+    * To bring your local database up to the latest version run:
+    ```
+    docker compose exec backend npx knex migrate:latest
+    docker compose exec backend npx knex seed:run
+    ```
+---
+
+5. Update the .env file in the backend directory ([backend .env](/backend/.env)) with the following:
+    
+   ```
+    NODE_ENV=development<br>
+    JWKSURI=https://oidc.gov.bc.ca/auth/realms/aaoozhcp/protocol/openid-connect/certs<br>
+    POSTGRES_PORT=15432<br>
+    POSTGRES_HOST=localhost<br>
+    POSTGRES_USER=postgres<br>
+    POSTGRES_PASSWORD=postgres<br>
+    POSTGRES_DATABASE=gat_db<br>
+   ```
+  ---
+
+## Start Servers
+1. Start the Database container in docker 
+    *   Open a terminal in the root of the project directory and run:
+         ```
+         docker compose start db
+         ```
+
+2. Start the backend and frontend
+    *   Open a terminal in the /frontend directory and run:
+        ```
+        npm run start
+        ```
+    *   Open a terminal in the /backend directory and run:
+         ```
+         npm run start
+         ```
+
+3. In your browser, visit localhost:3000 to access the site and login
+
+
```

Comment:
If you are running it this way, without the containers, where is the frontend configured to reach the backend, and on which port?